### DISCUSSION TITLE
Hello World

### DISCUSSION THREAD

**Discussion by Mikrizi** [MidnightBSD/src#179] [General]
Please add this place to the official website as a forum page, unite people using our OS

is it possible to port KDE from FreeBSD?

**Comment by laffer1** [comment]
We used to have KDE 3 and 4 in mports, but had a lot of problems with later versions.  The Qt5/6 and KDE ports don't tend to play nice with our fake/stage setup in mports for one thing.  There are a lot of path issues with libraries/includes.  

The steps involved would be:
1. update qt5
2. port the KDE build script stuff  (kf5-* ports) in freebsd ports/devel
3. make appropriate changes to the qt and kde.mk files in mports/Mk/extensions
4. start porting individual KDE ports in dependency order. 

I haven't seriously looked into it since kde 5 first came out.  

Short answer: it's likely possible but a fair amount of work. Would likely need to split up the work with a few people to do it reasonably soon.

**Reply by Mikrizi** [reply]
good afternoon, thanks for your answer, I also created an entry in the src section, I would be glad if you also pay attention to it.

your system is very promising and with a great idea, "domesticate" BSD so that every person can install a system for themselves by conditionally clicking next, next.

thank you for your work

**Comment by Roach445** [comment]
Hello, I wanna install midnightbsd on my late 2009 Macbook 6,1 does midnightbsd have support for the iSight camera?

**Reply by laffer1** [reply]
I haven't tested this.  If you're talking about a built in isight camera, those are typically usb 2.0 and will likely work with webcamd.  The original firewire one doesn't work afaik.

**Reply by Roach445** [reply]
Yeah I was talking about the built in iSight camera, so it would work? if it does ty.

**Comment by grahamperrin** [comment]
@laffer1 <https://forums.freebsd.org/posts/753089>: 

> … FreeBSD … is the only BSD that has an official forum.

Sigh …
